tests/DYCOMS2D/calc.cpp

```diff
@@ -31,32 +31,18 @@ int main(int ac, char** av)
       tmp << float(left_edges[i] / si::metres) << ":" << float(left_edges[i + 1] / si::metres) << "|0;";
     bins_wet_str = tmp.str();
   }
-/*  outdir=std::getenv("STORAGE");
-  string tmp=std::getenv("NODE_CONF");
-  outdir+="/"+tmp+"/out_lgrngn";
-  std::cout << "output directory: " << outdir << std::endl;
-*/
-  string opts_common = 
-    "--outfreq=600 --nt=7200 --spinup=3600 --nx=33 --nz=76 --dt=1 --relax_th_rv=false"; // DYCOMS: 128x300 ; 600 21600 3600
-  set<string> opts_micro({
-//    "--micro=blk_1m --outdir=out_blk_1m",
-//    "--micro=blk_2m --outdir=out_blk_2m",
-    "--adv_serial=false --async=true --micro=lgrngn --outdir=out_lgrngn --backend=CUDA --sd_conc=32 --sstp_cond=1 --z_rlx_sclr=100 --sstp_coal=1"  
-    " --coal=false"
-      " --out_wet=\""
```

Comment:
I think that in the Dycoms2D test with lagrangian micro, the out_wet option is needed to plot droplet size spectrum in plot_lgrngn_mean_spec.cpp and plot_lgrngn_spec.cpp.

Reply:
Also, to get dx=50m and dz=5m you need to set nx=129 and nz=301, because edge cells are half size (staggered grid).